doc/services/pm/device.rst

```diff
@@ -20,90 +27,153 @@ in power saving mode. This method allows saving power even when the CPU is
 appropriate power state by the application or component managing the devices.
 
+More information about this feature can be found in :ref:`Device Runtime PM <pm-device-runtime>`
+
+.. _pm-device-system-pm:
+
 System Power Management
 ***********************
 
 In this method device power management is mostly done inside
 :code:`pm_system_suspend()` along with entering a CPU or SOC power state.
 
-If a decision to enter a lower power state is made, the implementation would enter it
-only after checking if the devices are not in the middle of a hardware
-transaction that cannot be interrupted. This method can be used in
-implementations where the applications and components using devices are not
-expected to be power aware and do not implement runtime device power management.
+If a decision to enter a CPU lower power state is made, the power management
+subsystem will suspend devices before changing state. The subsystem takes care
+of suspend devices following their initialization order, ensuring that possible
+dependencies between them are satisfied. As soon as the CPU wakes up from a
+sleep states, devices are resumed following the reverse order that they were
+suspended.
 
-.. image:: images/central_method.svg
-   :align: center
+.. note::
+
+   When using :ref:`pm-system`, device transitions can be run from the idle thread.
+   As functions in this context cannot block, transitions that intend to use blocking
+   API's **must** check whether they can do so with :c:func:`k_can_yield`.
 
-This method can also be used to emulate a hardware feature supported by some
-SOCs which triggers automatic entry to a lower power state when all devices are idle.
-Refer to `Busy Status Indication`_ to see how to indicate whether a device is busy
-or idle.
+This method can be used in implementations where the applications and components
+using devices are not expected to be power aware and do not implement runtime
+device power management.
+
+.. note::
+
+    The CPU will enter a low power state only if the devices are not in the
+    middle of a hardware transaction that cannot be interrupted.
 
 Device Power Management States
 ******************************
 
 The power management subsystem defines device states in
-:c:enum:`pm_device_state`. These states are classified based on the degree of
-device context that gets lost in those states, kind of operations done to save
-power, and the impact on the device behavior due to the state transition. Device
-context includes device registers, clocks, memory etc.
+:c:enum:`pm_device_state`. This type is used to track power states of
+a particular device. It is important to emphasizes that, although the
+state is tracked by the subsystem, it is a device responsibility to
+handle an action (:c:enum:`pm_device_action`) to change its state.
 
-Device Power Management Operations
-**********************************
+Each :c:enum:`pm_device_action` have a direct an ambiguous relationship with
```

Comment:
??

Did you mean "has a direct and unambiguous" instead?

Reply:
yep :p